session/manager_http.go

```diff
@@ -214,7 +222,6 @@ func (s *ManagerHTTP) FetchFromRequest(ctx context.Context, r *http.Request) (_
 		return nil, errors.WithStack(NewErrNoActiveSessionFound())
 	}
 
-	se.Identity = se.Identity.CopyWithoutCredentials()
```

Comment:
`CopyWithoutCredentials()` is kinda old. We now have a custom JSON marshaller registered on identity which will omit credentials. That's a much safer way than making sure that we delete the credentials properly. It also meant that we could not optimize the whoami endpoint, as it requires the credentials (sometimes) to make a decision whether to request 2FA or not.